modules/examples/src/main/java/com/ibm/cloud/platform_services/iam_policy_management/v1/IamPolicyManagementExamples.java

```diff
@@ -0,0 +1,347 @@
+ * IAM_POLICY_MANAGEMENT_APIKEY= <YOUR_APIKEY>
+ * IAM_POLICY_MANAGEMENT_TEST_ACCOUNT_ID= <YOUR_ACCOUNT_ID>
+ *
+ * Alternatively, above envirnmonet variables can be placed in a "credentials" file and then:
+ * export IBM_CREDENTIALS_FILE=<name of credentials file>
+ *
+ */
+
+public class IamPolicyManagementExamples {
+  private static final Logger logger = LoggerFactory.getLogger(IamPolicyManagementExamples.class);
+  protected IamPolicyManagementExamples() { }
+
+  private static Random random = new Random();
+  private static final String TEST_UNIQUE_ID = String.valueOf(random.nextInt(100000));
+  private static final String TEST_USER_PREFIX = "IBMid-SDKJava";
+  private static final String TEST_USER_ID = TEST_USER_PREFIX + TEST_UNIQUE_ID;
+  private static final String TEST_SERVICE_NAME = "iam-groups";
+  private static String testAccountId = null;
+  private static String testPolicyId = null;
+  private static String testPolicyEtag = null;
+  private static String testCustomRoleId = null;
+  private static String testCustomRoleEtag = null;
+
+  static {
+    System.setProperty("IBM_CREDENTIALS_FILE", "../../iam_policy_management.env");
+  }
+
+  @SuppressWarnings("checkstyle:methodlength")
+  public static void main(String[] args) throws Exception {
+    IamPolicyManagement service = IamPolicyManagement.newInstance();
+
+    // Load up our test-specific config properties.
+    Map<String, String> config = CredentialUtils.getServiceProperties(IamPolicyManagement.DEFAULT_SERVICE_NAME);
+    testAccountId = config.get("TEST_ACCOUNT_ID");
+
+    try {
+      // begin-create_policy
+      ResourceAttribute resourceAttributeAccount = new ResourceAttribute.Builder()
+              .name("accountId")
+              .value(testAccountId)
+              .operator("stringEquals")
+              .build();
+
+      ResourceAttribute resourceAttributeService = new ResourceAttribute.Builder()
+              .name("serviceName")
+              .value(TEST_SERVICE_NAME)
+              .operator("stringEquals")
+              .build();
+
+      ResourceAttribute resourceAttributeResource = new ResourceAttribute.Builder()
+              .name("resource")
+              .value("SDK-Java-Test")
+              .operator("stringEquals")
+              .build();
+
+      ResourceTag resourceAttributeTag = new ResourceTag.Builder()
+              .name("project")
+              .value("prototype")
+              .operator("stringEquals")
+              .build();
+
+      SubjectAttribute subjectAttributeModel = new SubjectAttribute.Builder()
+              .name("iam_id")
+              .value(TEST_USER_ID)
+              .build();
+
+      PolicyResource policyResourceModel = new PolicyResource.Builder()
+              .attributes(new ArrayList<ResourceAttribute>(Arrays.asList(resourceAttributeAccount,
+                      resourceAttributeService, resourceAttributeResource)))
+              .tags(new ArrayList<ResourceTag>(Arrays.asList(resourceAttributeTag)))
```

Comment:
In this case, I like to use the generated "addXXX" method (e.g. addTags() in this case), because that lets you add a single tag value to the underlying list without all the syntactic sugar of constructing a new ArrayList, etc.
```suggestion
              .addTags(resourceAttributeTag)
```

Reply:
Much cleaner approach, thanks Phil. Updating the other cases where we do similar process